docs/usage/shoot/shoot_limits.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Shoot Cluster Limits
+---
+
+# Shoot Cluster Limits
+
+Gardener operators can configure limits for shoot clusters in the `CloudProfile.spec.limits` section, where they can also be looked up by shoot owners.
+The limits are enforced on all shoot clusters using the respective `CloudProfile`.
+If a certain limit is not configured, no limit is enforced.
+
+This document explains the limits that can be configured in the `CloudProfile`.
+
+## Maximum Node Count
+
+The `CloudProfile.spec.limits.maxNodesTotal` configures the maximum supported node count of shoot clusters in a Gardener installation.
+
+If this limit is set, Gardener ensures that
+
+- the total minimum node count of all worker pools (i.e., the total initial node count) does not exceed the configured limit
+- the maximum node count of an individual worker pool does not exceed the configured limit
+- cluster-autoscaler does not provision more nodes than the configured limit (`--max-nodes-total` flag)
+
+The maximum node count of a shoot cluster can be lower than the configured limit, if the cluster's networking configurations don't allow it (see [this doc page](../networking/shoot_networking.md)).
+
+Gardener operators must ensure that no existing shoot cluster exceeds the limit when adding it.
+Because Gardener itself cannot verify that all shoot clusters would comply with a given limit, it does not allow decreasing the limit, which could be disruptive for existing shoots.
```

Comment:
Why not? According to above list, it should be able to verify this, no?

Reply:
Well, I meant gardener-apiserver can't verify that a given limit would not disrupt shoots because it cannot determine how many nodes the clusters currently have.
In the list above, gardenlet configures the cluster-autoscaler's limit.
I clarified this sentence – hope this is better now.